src/os/bluestore/HMSMRDevice.h

```diff
@@ -0,0 +1,166 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2014 Red Hat
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+// Copied from KernelDevice with HM-SMR specific functionality added.  Will be
+// further specialized for HM-SMR.
+
+#ifndef CEPH_OS_BLUESTORE_HMSMRDEVICE_H
+#define CEPH_OS_BLUESTORE_HMSMRDEVICE_H
+
+#include <atomic>
+
+#include "include/types.h"
+#include "include/interval_set.h"
+#include "common/Thread.h"
+#include "include/utime.h"
+
+#include "ceph_aio.h"
+#include "BlockDevice.h"
+
+extern "C" {
+#include <libzbc/zbc.h>
+}
```

Comment:
no need to include zbc.h here. as we have `zbc_*()` calls only in the .cc file.

Reply:
Done.  Moved include into HMSMRDevice.cc and BlockDevice.cc.